v3/qwertykeys/qk100/qk100_ansi.json

```
@@ -0,0 +1,419 @@
+{
+  "name": "qk100 ansi",
+  "vendorId": "0x4F53",
+  "productId": "0x3030",
+  "keycodes": ["qmk_lighting"],
+  "menus": [
```

```suggestion
  "menus": [ "qmk_rgb_matrix" ],
```

you can replace the entire RGB block with this.

also you have rgb MATRIX enabled, not rgb LIGHT in your qmk firmware.